migrations/migrations/20210621204446-update-users-add-admin.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
     queryInterface.addColumn('users', 'role', {
       allowNull: false,
       type: Sequelize.INTEGER,
```

Comment:
as a suggestion sequelize has a type called enumSequelize that could be the most appropriate for this case